test-poc/framework/src/main/java/org/keycloak/test/framework/database/TestDatabase.java

```diff
@@ -79,6 +80,10 @@ private GenericContainer<?> createContainer() {
                 DockerImageName MSSQL = DockerImageName.parse(databaseConfig.getContainerImage()).asCompatibleSubstituteFor("sqlserver");
                 yield configureJdbcContainer(new MSSQLServerContainer<>(MSSQL));
             }
+            case OracleDatabaseSupplier.VENDOR -> {
+                DockerImageName ORACLE = DockerImageName.parse(databaseConfig.getContainerImage()).asCompatibleSubstituteFor("gvenzl/oracle-free");
```

Comment:
Did you consider our quay image?
- https://github.com/keycloak/keycloak/blob/main/testsuite/integration-arquillian/pom.xml#L507
This is necessary to have at least 1GB of db memory
- https://github.com/keycloak/keycloak/blob/main/testsuite/integration-arquillian/pom.xml#L509
This is a post step to initialiaze a schema and a user access with rights: https://github.com/keycloak/keycloak/blob/main/testsuite/integration-arquillian/pom.xml#L520

Reply:
We can't use images in private quay.io repos, that's why we use a public image in dockerhub. Just added the post start command

Reply:
Works for now, but not keen on using random containers. Might be better to publish our own if that is allowable by Oracle.

Can be done later though, but would at least create an issue around replacing this image?

Reply:
Not really following the fact that there's one image here, and another one in OracleDatabaseSupplier.

Reply:
Changed. Kept only the image in `OracleDatabaseSupplier`.